src/components/Rockets/rockets.css

```diff
@@ -0,0 +1,53 @@
+.flex-sm-row {
+  flex-direction: row !important;
+}
+
+.rounded {
+  border-radius: var(--bs-border-radius) !important;
+}
+
+.gap-3 {
+  gap: 1rem !important;
+}
+
+.p-3 {
+  padding: 1rem !important;
+}
+
+.mb-5 {
+  margin-bottom: 3rem !important;
+}
+
+.mb-4 {
+  margin-bottom: 1.5rem !important;
+}
+
+.align-items-center {
+  align-items: center !important;
+}
+
+.shadow {
+  box-shadow: 0 0.5rem 1rem rgba(0, 0, 0, 0.15) !important;
+}
+
+.d-flex {
+  display: flex !important;
+}
```

Comment:
- [ ] [OPTIONAL] The overuse of `!important` clause to enforce CSS values is really not the best as it is considered best practice to do without it or at least not beyond 3 times, as it could easily introduce bugs in the code, making the code quite messy, among other things :wink: